go/setup/integration/action.yaml

```diff
@@ -13,15 +17,19 @@ runs:
       with:
         go-version: "1.23.x"
 
+    - name: Install Poetry
+      uses: snok/install-poetry@v1.3.4
+
     - name: Install Python
       uses: actions/setup-python@v5
       with:
-        python-version: "3.x"
-
-    - name: Install setuptools
-      shell: bash
-      run: pip install setuptools
+        python-version: ${{ inputs.python-version }}
+        architecture: "x64"
+        cache: "poetry"
+        cache-dependency-path: |
+          toolkit/poetry.lock
```

Comment:
is this safe to assume this file exist?

Reply:
I know what you mean, what if `toolkit/poetry.lock` is missing, well it will simply ignore that non-existent file when computing the cache key.